### PR TITLE
gh-1890 removed "slim.tests" environment option in UploadFile that wa…

### PR DESCRIPTION
**Pull request by llvdl**
…s not used in tests

### CODE REVIEW

**Comment by dopesong** [comment]
You removed `slim.files`, not `slim.tests` :)

**Comment by JoeBengalen** [comment]
bc break

File: Slim/Http/UploadedFile.php [hunk 1]
```diff
@@ -75,21 +75,12 @@ class UploadedFile implements UploadedFileInterface
...
-     * Create a normalized tree of UploadedFile instances from the Environment.
-     *
-     * @param Environment $env The environment
-     *
-     * @return array|null A normalized tree of UploadedFile instances or null if none are provided.
+     * Parses the $_FILES global
+     * @return array A normalized tree of UploadedFile instances or null if none are provided.
      */
-    public static function createFromEnvironment(Environment $env)
+    public static function createFromGlobals()
```
Comment (akrabat) [inline Slim/Http/UploadedFile.php:81]: We can't change public method signatures or how they operate within the 3.x series.

**Comment by akrabat** [comment]
We can't remove `slim.files` as an application may be using it.

**Pull request closed without merging**